apps/src/code-studio/pd/workshop_dashboard/components/workshop_management.jsx

```diff
@@ -40,10 +41,16 @@ export class WorkshopManagement extends React.Component {
       let surveyBaseUrl;
 
       if (
-        [WorkshopTypes.local_summer, WorkshopTypes.teachercon].includes(props.subject)
-        && new Date(this.props.date).getFullYear() >= 2018
+          (
+            [WorkshopTypes.local_summer, WorkshopTypes.teachercon].includes(props.subject)
+            && new Date(this.props.date).getFullYear() >= 2018
+          ) ||
+          (
+            ['CS Discoveries', 'CS Principles'].includes(props.course)
+            && props.subject !== 'Code.org Facilitator Weekend'
+            && new Date(this.props.date) >= new Date(2018, 8, 1)
+          )
```

Comment:
I wonder if we should move this logic into the workshop model on the server and send the answer as a prop, i.e. `hasSurveyResults`?